### PR TITLE
fix: ListV2 flaky Tests (DefaultSelectedItem)

### LINKED ISSUE #22252: [Bug]: Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_defaultSelectedItem_spec.js
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_defaultSelectedItem_spec.js - failing in CI

### Steps To Reproduce

Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_defaultSelectedItem_spec.js
is failing in ci

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

self hosted

### PR DESCRIPTION
**Pull request by Tooluloope**
## Description
Fix Meta Hydration Tests in ListV2 : This is an actual regression and to be handled here https://github.com/appsmithorg/appsmith/issues/22534
FIx DefaultSelectedItem in Listv2

Fixes #22252

Media
> A video or a GIF is preferred. when using Loom, don’t embed because it looks like it’s a GIF. instead, just link to the video

## Type of change
- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?

- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by yatinappsmith — CHANGES_REQUESTED** [review]
Avoid using thirdparty api. Instead use TED

**Comment by Tooluloope** [comment]
> Workflow run: https://github.com/appsmithorg/appsmith/actions/runs/4735099721. Commit: `60be657`. The following are new failures, please fix them before merging the PR:
> 
> 1. cypress/integration/Regression_TestSuite/ServerSideTests/OnLoadTests/JSOnLoad1_Spec.ts
> 
> 
> To know the list of identified flaky tests - [Refer here](https://app.appsmith.com/applications/613868bedd7786286ddd4a6a/pages/63ec710e8e503f763651791a)

@yatinappsmith DefaultSelectedItem is passing.

Running the tests again

**Review by yatinappsmith — CHANGES_REQUESTED** [review]
To skip the test please add it cypress.json

**Review by yatinappsmith — DISMISSED** [review]

**Review by yatinappsmith — APPROVED** [review]

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_Meta_Hydration_ServerSide_spec.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ function verifyMultiDropdownValuesCount(count, page = 1) {
   });
 }
 
-describe("List widget v2 - meta hydration tests", () => {
+describe.skip("List widget v2 - meta hydration tests", () => {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_Meta_Hydration_ServerSide_spec.js:94]: Please also consider adding the reason for skipping it. To not loose track of the reason.

**Pull request merged**